README.md

```diff
@@ -528,6 +529,38 @@ filtered. Modifiable payload includes:
 notice[:params][:my_param] = 'foobar'
 ```
 
+### Custom exception attributes
+
+The library supports custom exception attributes. This is useful if you work
+with custom exceptions, which define non-standard attributes and you can't
+attach any additional data with help of the [`add_filter`](#airbrakeadd_filter)
+API due to the fact that the data isn't available at confuration time yet.
+
+In this case, you could define a special hook method on your exception called
+`#to_airbrake`. The method must return a Hash the keys of which must be a subset
+of the ones mentioned in the [`Notice#[]`](#notice--notice) API.
+
+```ruby
+class MyException
+  def initialize
+    @http_code = 404
+  end
+
+  # The library expects you to define this method. You must return a Hash,
+  # containing the keys you want to modify.
+  def to_airbrake
```

Comment:
When somebody defines a method with this name it's already clear why. It also must correspond to our API scheme, so if this was `to_hash`, I would expect it to return this instead: `{ http_code: 404 }`. But we need the `params` key, for example.